README.md

```diff
@@ -1 +1,96 @@
-https://hub.docker.com/r/microsoft/oms/
+# Trying the container solution pack for Microsoft Operations Management Suite
+
+## What is the Microsoft Operations Management Suite?
+
+The Microsoft Operations Management Suite (OMS) is a software-as-a-service offering from Microsoft that allows Enterprise IT to manage any hybrid cloud. It offers log analytics, automation, backup and recovery, and security and compliance.  Sign up for a free account at [http://mms.microsoft.com](http://mms.microsoft.com) or read more about here: [https://www.microsoft.com/en-us/server-cloud/operations-management-suite/overview.aspx](https://www.microsoft.com/en-us/server-cloud/operations-management-suite/overview.aspx)
+
+This container solution pack will generate a container which will runs OMS agent within. This is for Linux OS which has restriction in installing the Operations Management Suite Agent directly. However, it can also be used with other support Linux OS as well.
+
+## What can you do with the container solution pack?
+
+With this feature, you'll be able to:
+
+- See information about all container hosts in a single location
+- Know which containers are running, what image they're running, and where they're running
+- See an audit trail for actions on containers
+- Troubleshoot by viewing and searching centralized logs without remoting to the Docker hosts
+- Find containers that may be &quot;noisy neighbors&quot; and consuming excess resources on a host
+- View centralized CPU, memory, storage, and network usage and performance information for containers
+
+## Joining the private preview
+
+You must be a member of the private preview to use this feature. To join, drop us a line at OMSContainers@microsoft.com.
+
+### Supported Linux Operating Systems and Docker:
+- Docker 1.8 and above
+- An x64 version of Ubuntu, CoreOS(stable), Amazon Linux, SUSE 13.2, CentOS 7, or SLES 12
+
+## Setting up
+As a pre-requisite, docker must be running prior to this installation. If you have installed before running docker, please re-install OMS Agent. For more information about docker, please go to https://www.docker.com/.
+
+You have two choices for how to capture your container information. You can use OMS for all containers on a container host, or designate specific containers to send information to OMS.
+
+#### Settings on container host
+1. Edit docker.service to add the following:
+```
+[Service]
+...
+Environment="DOCKER_OPTS=--log-driver=fluentd --log-opt fluentd-address=localhost:25225"
+...
+```
+Make sure you add $DOCKER_OPTS in "ExecStart=/usr/bin/docker daemon" within your docker.service file.
+example)
+```
+[Service]
+Environment="DOCKER_OPTS=--log-driver=fluentd --log-opt fluentd-address=localhost:25225"
+ExecStart=/usr/bin/docker daemon -H fd:// $DOCKER_OPTS
+```
+2. Restart docker service.
```

Comment:
Look at the output, this shows up as 1. for each of the items. Apparently, when you use ```text``` notation, that resets the item counter (annoying).

I recommend that you use bullet items for this reason.